client/src/helpers/makeAnImageFromName.ts

```diff
@@ -0,0 +1,35 @@
+export const createImageFromInitials = (name: string) => {
+  if (name == null) return;
+  name = name[0];
+
+  const size = 500;
```

Comment:
Let's delete this and just the Material UI's Avatar component

Reply:
This will still help In my opinion! We can either use the image or user initial